src/commands/quote.ts

```diff
@@ -64,7 +64,7 @@ async function getCloeMessage(db: EntityManager) {
       : null
 
     const indiener = member
-      ? `${member.first_name}${member.primary_last_name_prefix ? ` ${member.primary_last_name_prefix}` : ""} ${
+      ? `${member.given_name || member.first_name}${member.primary_last_name_prefix ? ` ${member.primary_last_name_prefix}` : ""} ${
```

Comment:
js heeft een [nullish-coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) die beter, betrouwbaarder is dan `||`